components/scream/tests/coupled/dynamics_physics/homme_shoc_cld_p3_rrtmgp/CMakeLists.txt

```diff
@@ -63,17 +63,17 @@ configure_file(${SCREAM_SRC_DIR}/dynamics/homme/tests/theta.nl
 GetInputFile(init/${EAMxx_tests_IC_FILE_72lev})
 
 ## Finally compare all MPI rank output files against the single rank output as a baseline, using CPRNC
-## Only if max mpi ranks is >1
+## Only if running with 2+ ranks configurations
 # This test requires CPRNC
-if (SCREAM_TEST_MAX_RANKS GREATER 1)
+if (TEST_RANK_START GREATER TEST_RANK_END)
```

Comment:
I see the following code pattern was repeated in at least 6 other places. Might be nice to encapsulate.